util/features.py

```diff
@@ -218,9 +245,13 @@ def design_features(
     Returns:
       pd.DataFrame of features
     """
-    # for soil, we need to comput water holding capacity
+    # for soil, we need to compute water holding capacity
     # TODO: 1. not needed for cybench data. Remove later.
     # TODO: 2. Add code to make drainage class a categorical feature.
+
+    if 'DRAIN' in soil_df.columns:
+        soil_df['DRAIN'] = soil_df['DRAIN'].astype('category')
```

Comment:
The sample data from NL does not yet have "drainage_class". The one now uploaded in Google drive has.